pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1125,4 +1125,6 @@ self: super: {
   # Tests require a browser: https://github.com/ku-fpg/blank-canvas/issues/73
   blank-canvas = dontCheck super.blank-canvas;
   blank-canvas_0_6_2 = dontCheck super.blank-canvas_0_6_2;
+
+  language-ecmascript = doJailbreak super.language-ecmascript;
```

Comment:
Please add a comment here why the jailbreak is needed. Something like:
```
# Needed because of testing-feat >=0.4.0.2 && <1.1
```

With that it's easier to know when we can remove the override again.